mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -228,12 +231,45 @@ def get_spectrum(self, roi: str | SensibleROI, mode: SpecType) -> np.ndarray:
                 return np.array([])
             roi_spectrum = self.get_stack_spectrum(self._stack, roi)
             roi_norm_spectrum = self.get_stack_spectrum(self._normalise_stack, roi)
-        return np.divide(roi_spectrum, roi_norm_spectrum, out=np.zeros_like(roi_spectrum), where=roi_norm_spectrum != 0)
+            average_shuttercount = self.get_shuttercount_normalised_correction_parameter()
```

Comment:
This is a stray line, its not needed as there  the same at 240 in the if statement